debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java

```diff
@@ -329,6 +329,33 @@ public static void waitForSnapshotToBeCompleted() throws InterruptedException {
         }
     }
 
+    public static void waitForStreamingRunning() throws InterruptedException {
+        int waitForSeconds = 60;
+        final MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
+        final Metronome metronome = Metronome.sleeper(Duration.ofSeconds(1), Clock.system());
+
+        while (true) {
```

Comment:
Did you consider to use Awaitility? Would be more in line with what we do elsewhere in similar cases.

Reply:
I did not, but I can certainly modify the PR to use it instead.  I'll push an update shortly.